makepanda/makewheel.py

```diff
@@ -0,0 +1,195 @@
+"""
+Generates a wheel (.whl) file from the output of makepanda.
+
+Since the wheel requires special linking, this will only work if compiled with
+the `--wheel` parameter.
+"""
+from __future__ import print_function, unicode_literals
+from distutils.util import get_platform as get_dist
+import json
+
+import sys
+import os
+from os.path import join
+import shutil
+import zipfile
+from makepandacore import ColorText
+
+
+def get_platform():
+    p = get_dist().replace('-', '_').replace('.', '_')
+    if "linux" in p:
+        print(ColorText("red", "WARNING:") +
+              " Linux-specific wheel files are not supported."
+              " We will generate this wheel as a generic package instead.")
+        return "any"
+    return p
+
+
+# Global filepaths
+LIB_SRC = join("built", "lib")  # TODO: A special wheel lib dir
```

Comment:
The "built" directory should be a parameter to the makewheel function.  On 64-bit Windows it is called "built_x64" and we actually allow overriding this (which I do a lot, and so does the buildbot) via the --outputdir parameter.

See makepandacore.GetOutputDir()